docs/source/deployorderer/ordererchecklist.md

```diff
@@ -0,0 +1,307 @@
+# should be changed to the real location in production.
+LocalMSPDir: msp
+```
+
+**`LocalMSPDir`**: (default value should not be overridden) If you change this from `msp`, your folder structure must reflect the folder name you have specified here. The pathname is relative to the `FABRIC_CFG_PATH` or you can specify an absolute path.
+
+## General.LocalMSPID
+
+```
+# LocalMSPID is the identity to register the local MSP material with the MSP
+# manager. IMPORTANT: The local MSP ID of an orderer needs to match the MSP
+# ID of one of the organizations defined in the orderer system channel's
+# /Channel/Orderer configuration. The sample organization defined in the
+# sample configuration provided has an MSP ID of "SampleOrg".
+LocalMSPID: SampleOrg
+```
+
+**`LocalMSPID`**: (default value should be overridden) The MSP ID must match the orderer organization MSP ID that exists in the configuration of the system channel. This means the MSP ID must have been listen in the `configtx.yaml` used to create the genesis block of the system channel (or have been added later to the list of system channel administrators).
+
+## General.BCCSP.*
+
+BCCSP.SW can’t be configured at the moment. Leave those parameters as they are.
+Should this section be similar to the same content in the  CA and Peer sections?
+
+**`PKCS11`**:
+
+```
+# Default specifies the preferred blockchain crypto service provider
+        # to use. If the preferred provider is not available, the software
+        # based provider ("SW") will be used.
+        # Valid providers are:
+        #  - SW: a software based crypto provider
+        #  - PKCS11: a CA hardware security module crypto provider.
+        Default: SW
+
+        # SW configures the software based blockchain crypto provider.
+        SW:
+            # TODO: The default Hash and Security level needs refactoring to be
+            # fully configurable. Changing these defaults requires coordination
+            # SHA2 is hardcoded in several places, not only BCCSP
+            Hash: SHA2
+            Security: 256
+            # Location of key store. If this is unset, a location will be
+            # chosen using: 'LocalMSPDir'/keystore
+            FileKeyStore:
+                KeyStore:
+```
+
+(Optional) This section is used to configure the Blockchain crypto provider.
+
+- **`BCCSP.Default:`** If you plan to use a Hardware Security Module (HSM), then this must be set to `PKCS11`.
+- **`BCCSP.PKCS11.*:`** Provide this set of parameters according to your HSM configuration. Refer to this [example]((../hsm.html) of an HSM configuration for more information.
+
+## FileLedger.Location
+
+```
+# Location: The directory to store the blocks in.
+Location: /var/hyperledger/production/orderer
+```
+
+**`Location`**: (default value should be overridden, especially if more than node will be created by this organization) Every channel on which the node is a consenter will have its own subdirectory at this location.
```

Comment:
The user running the orderer needs to own and have write access to this directory.